net/nut/files/nut-monitor.init

```diff
@@ -159,6 +159,11 @@ start_service() {
 	chmod 640 $UPSMON_C
 	chmod 640 /var/etc/nut/nut.conf
 
+	[ -d /var/run/nut ] || {
+		mkdir -m 0750 -p /var/run/net
```

Comment:
On Mon, 2016-07-04 at 06:03 -0700, Yousong Zhou wrote:
> In net/nut/files/nut-monitor.init:
> > @@ -159,6 +159,11 @@ start_service() {
> >  	chmod 640 $UPSMON_C
> >  	chmod 640 /var/etc/nut/nut.conf
> >  
> > +	[ -d /var/run/nut ] || {
> > +		mkdir -m 0750 -p /var/run/net
> Typo in /var/run/net?

I think this one didn't trigger because I was also running nut-server
which already created it.

Thanks,

Daniel